excel-importer-product/README.md

```diff
@@ -31,5 +30,15 @@ In the project, where the Excel data should be managed:
 1. Create a persistence unit under `/config/persistence.xml`
 2. Add the properties
   - `hibernate.hbm2ddl.auto=update` (to allow schema changes)
-  - `hibernate.id.new_generator_mappings=false` (to use classic sequence)
```

Comment:
this is no longer necessary with #80, as we now explicitly define our sequence-identity on the 'id' field.